README.md

```diff
@@ -22,9 +24,12 @@ Add the following snippet to the beginning of the `<head>` tag within your marku
 
 The above snippet will fetch the Raygun4JS script from our CDN asynchronously, so it doesn't block other scripts from being loaded. It will also catch errors that are thrown while the page is loading, and send them when the script is ready.
 
+### Via package manager installation 
+For installations and usage via a package manager, refer to the [Synchronous methods](#synchronous-methods) section of this document.
+
 **Step 2**
 
-Add the following lines to your JS site code and paste in your API key (from your Raygun dashboard), to set up the provider to automatically send errors to your Raygun:
+Add the following lines to your JS site code and paste in your API key (from your Raygun dashboard), to set up the provider to automatically send errors to your Raygun app:
```

Comment:
Maybe say from Application Settings or maybe "installation page" since "Dashboards" is a different thing which people may be confused about